lib/improver/blending/blend_across_adjacent_points.py

```diff
@@ -90,10 +98,10 @@ def __init__(self, coord, width, parameter_units, weighting_mode):
     def __repr__(self):
         """Represent the configured plugin instance as a string."""
         msg = ('<TriangularWeightedBlendAcrossAdjacentPoints:'
-               ' coord = {0:s}, width = {1:.2f},'
-               ' parameter_units = {2:s}, mode = {3:s}>')
-        return msg.format(self.coord, self.width, self.parameter_units,
-                          self.mode)
+               ' coord = {0:s}, central_point = {1:.2f}, '
+               'parameter_units = {2:s}, width = {3:.2f}, mode = {4:s}>')
+        return msg.format(self.coord, self.central_point, self.parameter_units,
+                          self.width, self.mode)
 
     @staticmethod
     def correct_collapsed_coordinates(orig_cube, new_cube, coords_to_correct):
```

Comment:
I'm not sure this is what we want to do. I think we want to make a copy of the extracted cube at the central point and put the new data in this cube. This will retain whatever bounds are on the input cube.